scan/src/components/ConnectWithLedger.re

```diff
@@ -145,8 +152,10 @@ let make = () => {
        | Nothing => React.null
        }}
     </div>
-    <div className=Styles.connectBtn onClick={_ => createLedger()}>
-      <Text value="Connect To Ledger" weight=Text.Bold size=Text.Md color=Colors.white />
+    <div className={Styles.connectBtn(result == Loading)} onClick={_ => createLedger()}>
+      {result == Loading
```

Comment:
There are two duplicate of "result == Loading"
Should it be reduced to just one?